From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Mykyta Komarnytskyy <nkomarn@hotmail.com>
Date: Wed, 7 Oct 2020 15:23:17 -0700
Subject: [PATCH] Add server config


diff --git a/src/main/java/org/bukkit/Server.java b/src/main/java/org/bukkit/Server.java
index 7a51edbde474bf610cb5928de4b1bbe2edf34638..c2b8c2d3953e77bfc575ec92837219a23b66267f 100644
--- a/src/main/java/org/bukkit/Server.java
+++ b/src/main/java/org/bukkit/Server.java
@@ -1483,6 +1483,14 @@ public interface Server extends PluginMessageRecipient {
         }
         // Tuinity end - add config to timings report
 
+        // Hydrinity start - Server config
+        @NotNull
+        public org.bukkit.configuration.file.YamlConfiguration getHydrinityConfig()
+        {
+            throw new UnsupportedOperationException("Not supported yet.");
+        }
+        // Hydrinity end - Server config
+
         /**
          * Sends the component to the player
          *
